config/vufind/config.ini

```diff
@@ -840,6 +847,14 @@ authors         = Wikipedia
 url = "https://api.vlb.de/api/v1/cover/"
 ; token = "XXXXXX-XXXX-XXXXX-XXXXXXXXXXXX"
 
+; This section is needed for LocalFile cover loading.
+; Note:  All coverimage LocalFile:ConfigSection must include the following keys: 
+;        filepath, filename, and imagetype.
+[LocalFile]
+filepath = /usr/local/vufind/local/coverimages
+filename = recordid
```

Comment:
If we go with a multivalued approach, would it make more sense to just have one multivalued field in which we specify the entire filepath (inclusive of the base directory path, filename structure and image type)? Then we just need one configuration section which allows for multiple paths to be specified in the filename[] array. What do you think?